sanitiser/_geo_common.js

```diff
@@ -55,6 +56,7 @@ function sanitize_rect( key_prefix, clean, raw, bbox_is_required ) {
  * @param {bool} circle_is_required
  */
 function sanitize_circle( key_prefix, clean, raw, circle_is_required ) {
+  // "boundary.circle", clean, raw, false
```

Comment:
the info in this comment should be moved to the doc block above the function definition.